rpc/src/eth.rs

```diff
@@ -358,10 +398,8 @@ impl<B, C, P, CT, BE, A> EthApiT for EthApi<B, C, P, CT, BE, A> where
 			return Ok(None);
 		}
 
-		let block = self.client.runtime_api().current_block(&id)
-			.map_err(|err| internal_err(format!("call runtime failed: {:?}", err)))?;
-		let statuses = self.client.runtime_api().current_transaction_statuses(&id)
-			.map_err(|err| internal_err(format!("call runtime failed: {:?}", err)))?;
+		let block: Option<ethereum::Block> = self.current_block(&id);
+		let statuses: Option<Vec<TransactionStatus>> = self.current_statuses(&id);
```

Comment:
Why explicit type signatures?

Reply:
And why remove this error message? That could be really useful for someone who hits a bug in the future.

Reply:
> Why explicit type signatures?

Readibility.

> And why remove this error message? That could be really useful for someone who hits a bug in the future.

We can add them!